### ISSUE TITLE
Issue on page /tools-for-mathematics/03-calculus/tutorial/main.html

### ISSUE THREAD

**Issue by certik** [drvinceknight/pfm#80]
I clicked on the Binder button at the top right. The binding kept launching as usual, it build the Docker image etc., and after about 2 minutes it redirected me to this page:

![Screenshot from 2020-10-09 15-17-57](https://user-images.githubusercontent.com/20568/95632411-c9852e80-0a42-11eb-99fa-fcc24d7d11dc.png)

Not sure if this is a bug in Binder or your page. I used Chrome on Ubuntu 18.04.

**Comment by drvinceknight** [comment]
Yeah I get the same, I actually tried to figure out how to remove that button all together but I couldn't figure out how to do that in the jupyterbook config.

I **assume** the error is on my side in that I've not setup a binder build.

Thanks for opening the issue, will keep it open until I get to this (but it's not high priority for me at the moment).

Really appreciate you taking the time to look at the book (❤️ `sympy`).
